docs/internals.md

```diff
@@ -1,12 +1,70 @@
-# Data storage
+# Internals
 
-This page describes how DAMNIT saves the data it creates. This will probably not
-be of interest to you unless you're developing DAMNIT or writing tools that read
-data created by DAMNIT.
+This page describes some internal details of DAMNIT. It will probably not be of
+interest to you unless you're a developer.
 
----
+## Installation
```

Comment:
I'd say no because I anticipate making API releases fairly infrequently, whereas if someone's installing the frontend/backend then they probably want the latest version.